specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Trigger.json

```diff
@@ -324,6 +324,13 @@
             "retryPolicy": {
               "$ref": "#/definitions/RetryPolicy",
               "description": "Retry policy that will be applied for failed pipeline runs."
+            },
+            "dependsOn": {
+              "type": "array",
+              "description": "Tumbling window triggers that this trigger depends on.",

```

Comment:
Leave the description here the same to specifically indicate that it is a tumbling window trigger

Reply:
You can also add - "Only tumbling window triggers are supported." If we ever add other trigger types here we will update the description.